README.md

```diff
@@ -1,4 +1,7 @@
 # Qwist postman collections
 
 Postman collections for every Qwist product. 
-You can use these to easily try out different products provided by Qwist (with little to no effort).
+You can use them to easily try out different products provided by Qwist (with little to no effort).
+
+
+### NOTE: You will need Qwsit client's credentials to access the API. You can get them from your contact person with Qwist or via the form [qwist.com/contact](https://qwist.com/contact)
```

Comment:
You will need client credentials to access the API. Either request them from your contact person or via the [form](url).